packages/recode/build.sh

```diff
@@ -7,3 +7,10 @@ TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/rrthomas/recode/releases/download/v${TERMUX_PKG_VERSION}/recode-${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=4fb75cacc7b80fda7147ea02580eafd2b4493461fb75159e9a49561d3e10cfa7
 TERMUX_PKG_DEPENDS="libiconv"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_MAKE_PROCESSES=1
```

Comment:
What error did you get without this?

Reply:
It keeps result in random linking error (different each time) so i tried this and it worked.